backend/src/models/publisher.ts

```diff
@@ -0,0 +1,17 @@
+import mongoose, { Schema, Document } from 'mongoose';
+
+export interface PublisherEntity extends Document {
+    name: string;
```

Comment:
To `_id` to albo wszędzie, albo nigdzie i albo z `?` albo bez bo tak trochę dziwnie że w każdym pliku inaczej